### PR TITLE
Scan by method

### PR DESCRIPTION
**Pull request by Aharonyn**

### CODE REVIEW

File: django_mri/models/managers/scan.py [hunk 1]
```diff
@@ -21,3 +22,20 @@ def import_path(
     ) -> tuple:
         dicom_scans = self.import_dicom_data(path, progressbar, report)
         return {ScanType.DICOM.value: dicom_scans}
+
+    def scan_by_method(self, method: str) -> list:
```
Comment (ZviBaratz) [inline django_mri/models/managers/scan.py:26]: `by_method`? This seems to simply apply a `SequenceType` based filtering, doesn't it?

File: django_mri/models/managers/scan.py [hunk 2]
```diff
@@ -21,3 +22,20 @@ def import_path(
...
+            title__icontains=method
+        ).sequence_definitions
+        sequence_variants = map(
+            lambda definition: definition["sequence_variant"],
+            mri_method_definitions,
+        )
+        scanning_sequences = map(
+            lambda definition: definition["scanning_sequence"],
+            mri_method_definitions,
+        )
```
Comment (ZviBaratz) [inline django_mri/models/managers/scan.py:37]: This might be more readable using list comprehensions.

**Review by ZviBaratz — COMMENTED** [review]
Looks good, see what you think about the two minor comments, otherwise approved.

File: django_mri/models/managers/scan.py (hunk 2 above)
Comment (Aharonyn) [reply django_mri/models/managers/scan.py:37]: Corrected.

File: django_mri/models/managers/scan.py (hunk 1 above)
Comment (Aharonyn) [reply django_mri/models/managers/scan.py:26]: It is, so how should we call it?

Comment (Aharonyn) [reply django_mri/models/managers/scan.py:26]: I will leave it with this name for now, we can change it later if it's problematic.

**Pull request merged**